python/private/python_bootstrap_template.txt

```diff
@@ -461,6 +461,9 @@ relative_files = True
     kparams['stdout'] = sys.stderr
     kparams['stderr'] = sys.stderr
 
+  # Prevent coveragepy report issues when packages execute files in tmp dir.
+  params.append('--omit="$TMPDIR/*"')
```

Comment:
Ref: https://github.com/nedbat/coveragepy/issues/1921#issuecomment-2629154773

I can confirm that passing `--omit="$TMPDIR/*"` would have coverage.py interpolate the env var correctly. I can reproduce this outside of Bazel.

However, when having it inside of Bazel, it doesn't seem to work _(still getting the "No source for code: ..." error)_. Even if I do:

```python
tmpdir = os.getenv('TMPDIR')
if tmpdir:
    params.extend(['--omit', f'{tmpdir}/*'])
```

File pattern seems to be correct as well: https://coverage.readthedocs.io/en/7.6.10/source.html#file-patterns.

Also tried including the env vars `GTEST_TMP_DIR` and `TEST_TMPDIR` but to no avail.

I might be missing some other intermediate tmp dir that Torch utilizes when inside Bazel. 🤔

Reply:
Fixed by @rickeylev : https://github.com/bazelbuild/rules_python/pull/2599/commits/91d6cbede1629383ee479ccbc866dd0106e35f7c